narwhals/_spark_like/expr.py

```diff
@@ -387,13 +394,11 @@ def _log(expr: Column) -> Column:
 
         return self._with_elementwise(_log)
 
-    def sqrt(self) -> Self:
-        def _sqrt(expr: Column) -> Column:
-            return self._F.when(expr < 0, self._F.lit(float("nan"))).otherwise(
-                self._F.sqrt(expr)
-            )
+    def exp(self) -> Self:
+        def _exp(expr: Column) -> Column:
+            return self._F.exp(expr)
```

Comment:
not sure why `exp` is being added back here, perhaps an error while resolving merge conflicts?